x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/rules_tables.tsx

```diff
@@ -5,71 +5,59 @@
  * 2.0.
  */
 
-/* eslint-disable complexity */
-
 import {
   EuiBasicTable,
```

Comment:
Have you looked into https://elastic.github.io/eui/#/tabular-content/in-memory-tables ?
Is it something that might be helpful for our case? Especially this bit:

> It takes the full set of data (all possible items) and based on its configuration, will display it handling all configured functionality (pagination and sorting) for you.

Reply:
Good find, but I don't think it could help us. We implement sorting logic (see `getRulesComparator`) differently than EUI does. Even if we manage to align sorting algorithms, filtering still has to be done on our side. So sorting and filtering responsibilities will be spread among different components. And I think it is much better to keep that logic in one place, considering we need to implement a feature switch on top of it.